chorus_lib/src/core.rs

```diff
@@ -15,9 +15,9 @@ pub trait ChoreographyLocation: Copy {
 }
 
 /// Represents a value that can be used in a choreography. ChoRus uses [serde](https://serde.rs/) to serialize and deserialize values.
-/// It can be derived using `#[derive(Serialize, Deserialize, Clone)]` as long as all the fields satisfy the `ChoreographicValue` trait.
-pub trait ChoreographicValue: Serialize + DeserializeOwned + Clone {} // TODO(shumbo): Is `Clone` really necessary?
-impl<T: Serialize + DeserializeOwned + Clone> ChoreographicValue for T {}
+/// It can be derived using `#[derive(Serialize, Deserialize)]` as long as all the fields satisfy the `Portable` trait.
+pub trait Portable: Serialize + DeserializeOwned {}
```

Comment:
With this PR, we can have non-portable located values inside choreographies. Therefore, `ChoreographicValue` is misleading. `Portable` means the value is serializable + deserializable = can be move from one location to another.